ModelarDB-Analyze-Storage/main.py

```diff
@@ -1,3 +1,5 @@
+"""Script for analyzing how ModelarDB stores compress different data sets."""
```

Comment:
Should this be "Script for analyzing how ModelarDB stores different data sets."?

Reply:
Fixed.